### ISSUE TITLE
ClipartConfiguration color changes are not handled

### ISSUE THREAD

**Issue by AnEnglishmanInNorway** [bash0/cewe2pdf#63]
This is **definitely** a detail but I might as well record it. In my mcf file I have used a simple clipart (an arrow) and changed its color to black. We don't act on the color change
![image](https://user-images.githubusercontent.com/12747177/103782402-b4b33980-5037-11eb-87ed-9b646b4581fb.png)

**Comment by BarchSteel** [comment]
Addressed this in da064781f1f2f5c8102d0e5bc8eb291ad35305f3

**Comment by AnEnglishmanInNorway** [comment]
Thanks! I took your improvement, added simple parsing of the color changes in the mcf xml to provide it with the right parameters, and I now have a version which I could make available to fix this issue. 
However ... I forgot that I had pull request 73 outstanding and I've mixed these changes into that. I'd left 73 open in the hope that somebody might comment on it - either to object or to agree. I know that tidying code is often regarded as a nuisance because it creates "unnecessary" changes in the code which can be worrying when merging versions, and I'm therefore quite happy not to merge code tidying if anybody objects. Or shall I just go ahead?

**Comment by BarchSteel** [comment]
> Thanks! I took your improvement, added simple parsing of the color changes in the mcf xml to provide it with the right parameters, and I now have a version which I could make available to fix this issue.

I though the parsing of colors from XML in mcf was working, but maybe you mean the error handling of the clip-art files? I can't see a commit in your repository for it.

> However ... I forgot that I had pull request 73 outstanding and I've mixed these changes into that. I'd left 73 open in the hope that somebody might comment on it - either to object or to agree. I know that tidying code is often regarded as a nuisance because it creates "unnecessary" changes in the code which can be worrying when merging versions, and I'm therefore quite happy not to merge code tidying if anybody objects. Or shall I just go ahead?

I'm fine with the linter.

**Comment by AnEnglishmanInNorway** [comment]
Sorry, just committed the changes which now appear as part of the pull request https://github.com/bash0/cewe2pdf/pull/73. As you'll see, I have extended _processAreaClipartTag_ to pick out the color change configuration from the mcf, and then passed the result to _replaceColors_ after the clipart file has been loaded. I'll merge the pull request. And I guess I'll have to learn how to use git separate different logical changes onto different branches - I've used "traditional" VCS all my working life and am learning git as I go.
